docs/UserGuide.md

```diff
@@ -73,74 +73,99 @@ Shows a message explaning how to access the help page.
+* `book list` followed by `book delete 2` deletes the 2nd book from LibTask's database.
 
-### Deleting a person : `delete`
+### Borrowing a book : `borrow`
 
-Deletes the specified person from the address book.
+Establishes a relationship that patron at index INDEX1 borrows a book at index INDEX2.
 
-Format: `delete INDEX`
+Format: `borrow INDEX1 INDEX2`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* `INDEX1` refers to the index number shown in the displayed patron list.
+* `INDEX2` refers to the index number shown in the displayed book list.
+* `INDEX1` **must be a positive integer** 1, 2, 3, … and smaller than or equal to the number of patrons in the displayed list.
+* `INDEX2` **must be a positive integer** 1, 2, 3, … and smaller than or equal to the number of books in the displayed list.
+* The book at `INDEX2` must not be already borrowed.
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `patron list` and `book list` followed by `borrow 2 3` establishes a relationship that the 2nd patron borrows the 3rd book.
+
+### Returning a book : `return`
+
+Depending on the exact command, return all books borrowed by a patron at the specified index, or return a book at the specified index.
+
+Format: `return PREFIX/INDEX`
+
+* `PREFIX` must be either `p` for patrons or `b` for books.
+* If `PREFIX` is `p`, `INDEX` refers to the index number shown in the displayed patron list.
+* If `PREFIX` is `b`, `INDEX` refers to the index number shown in the displayed book list.
+* `INDEX` **must be a positive integer** 1, 2, 3, … and smaller than or equal to the number of patrons or books in the displayed list.
+* If the book at index `INDEX` is not borrowed, or if the patron at index `INDEX` does not borrow any books, nothing happens.
+
+Examples:
+* `patron list` followed by `return p/3` will return all books borrowed by the 3rd patron, if any.
+* `book list` followed by `return b/2` will return the 2nd book, if it is borrowed.
+
+### Requesting a book : `request`
+
+Establishes a relationship that patron at index INDEX1 is requesting to be notified when the book at index INDEX2 is available.
+
+Format: `request INDEX1 INDEX2`
+
+* `INDEX1` **must be a positive integer** 1, 2, 3, … and smaller than or equal to the number of patrons in the displayed list.
+* `INDEX2` **must be a positive integer** 1, 2, 3, … and smaller than or equal to the number of books in the displayed list.
+* If the book at index `INDEX2` is currently available, a message will be displayed.
+
+Examples:
+* `patron list` and `book list` followed by `request 1 2` keeps a record that the 1st patron would like to be notified when the 2nd book is available.
```

Comment:
The formatting of the patron features and details are consistent with our shared document.